pubspec.yaml

```diff
@@ -29,6 +29,7 @@ flutter:
   uses-material-design: true
   assets:
     - assets/logo.png
+    - assets/warning.png
```

Comment:
@LambrosKolovos warning.png is missing from git

Reply:
@sakrifor Should be fine now